clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -388,6 +389,24 @@ public class VenicePushJob implements AutoCloseable {
   public static final int NOT_SET = -1;
   private static final Logger LOGGER = LogManager.getLogger(VenicePushJob.class);
 
+  /**
+   * Config to enable single targeted region push mode in VPJ.
+   * In this mode, the VPJ will only push data to a single colo/region first, perform validation and finally
+   * leverage data recovery/repush to propagate data globally.
+   * The single solo/region is decided by the store config in {@link StoreResponse#getStore()}}.
+   * For multiple targeted regions push, may use the advanced mode. See {@link #TARGETED_REGIONS_PUSH}.
+   */
+  public static final String SINGLE_REGION_PUSH = "single.region.push";
```

Comment:
can we make name like "enable.single.region.push" or "single.region.push.enabled" ?

Reply:
Hmm, I'd like to keep it this way because I want both configs follow the same format as "xxx.xxx.push
e.g. `single.region.push` and `targeted.regions.push`. Lmk if you still prefer to change it.